rebasehelper/constants.py

```diff
@@ -20,8 +20,32 @@
 # Authors: Petr Hracek <phracek@redhat.com>
 #          Tomas Hozza <thozza@redhat.com>
 
+import os
+
 PROGRAM_DESCRIPTION = "Tool to help package maintainers rebase their packages to the latest upstream version"
 NEW_ISSUE_LINK = 'https://github.com/rebase-helper/rebase-helper/issues/new'
 
+REBASE_HELPER_RESULTS_DIR = "rebase-helper-results"
+REBASE_HELPER_WORKSPACE_DIR = "rebase-helper-workspace"
+REBASE_HELPER_REBASED_SOURCES_DIR = os.path.join(REBASE_HELPER_RESULTS_DIR, "rebased-sources")
+
+REBASE_HELPER_LOGS = "logs"
+REBASE_HELPER_DEBUG_LOG = 'debug.log'
+REBASE_HELPER_REPORT_LOG = 'report.log'
+REBASE_HELPER_LOGS_DIR = os.path.join(REBASE_HELPER_RESULTS_DIR, REBASE_HELPER_LOGS)
+
+OLD_SOURCES = 'old_sources'
+NEW_SOURCES = 'new_sources'
+OLD_SOURCES_DIR = os.path.join(REBASE_HELPER_WORKSPACE_DIR, OLD_SOURCES)
+NEW_SOURCES_DIR = os.path.join(REBASE_HELPER_WORKSPACE_DIR, NEW_SOURCES)
+
+GIT_CONFIG = '.gitconfig'
+
+BEGIN_COMMENT = '#BEGIN THIS MODIFIED BY REBASE-HELPER'
+END_COMMENT = '#END THIS MODIFIED BY REBASE-HELPER'
```

Comment:
This is not exactly in sope of this PR, but I'll mention it here. I'm not sure if the code that inserts these strings is actually working, but in any case, the text is kind of weird. I think something like `# BEGIN rebase-helper MODIFIED BLOCK` would be more appropriate.

Reply:
Good point, the comment messages don't make sense. I will have a look at the code using these constants and see if it works. If it doesn't, I will leave it for another PR and fix it